config.exit.json

```diff
@@ -4,7 +4,7 @@
   "ListenUDP": 30011,
   "Reverse": false,
   "ReverseRandomPorts": true,
-  "ReverseMaxPrice": "0",
+  "ReverseMaxPrice": "0.001",
```

Comment:
It's better to use 0 as default price so people who just generate wallet can try with the free nodes we provide

Reply:
The default price in entry config is 0.001, shoule we change the price to 0 together?

Reply:
We don't need to change provider side because provider wants to earn money (default price > 0), while user wants to use (default price = 0)

Reply:
Good point, it's better to keep them the same

Reply:
Let's keep 0.001 for both of them then